spec/support/api/schemas/artists.json

```diff
@@ -1,42 +1,89 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "id": "http://spotify-apprentice-app.com",
+  "id": "http://jsonschema.net",
```

Comment:
If you set the URL to `http://spotify-apprentice-app.com` on the schema generator site, you can set the URL throughout your schemas